includes/assets.php

```diff
@@ -71,10 +71,10 @@ function wpum_load_scripts() {
 add_action( 'wp_enqueue_scripts', 'wpum_load_scripts' );
 
 function wpum_enqueue_scripts() {
```

Comment:
:no_entry_sign: **Error**: Missing doc comment for function wpum_enqueue_scripts() (*Squiz.Commenting.FunctionComment.Missing*).